File: app/src/main/java/pedroluiz/projeto/soccernews/data/remote/SoccerNewsApi.java
package pedroluiz.projeto.soccernews.data.remote;

import java.util.List;

import pedroluiz.projeto.soccernews.domain.News;
import retrofit2.Call;
import retrofit2.http.GET;

public interface SoccerNewsApi {
    @GET("news.json")
    Call<List<News>> getNews();
}

File: app/src/main/java/pedroluiz/projeto/soccernews/data/remote/SoccerNewsApi.kt
package pedroluiz.projeto.soccernews.data.remote

import pedroluiz.projeto.soccernews.data.model.api.NewsResponse
import retrofit2.http.GET

interface SoccerNewsApi {
    @GET("news.json")
    suspend fun getListNews(): List<NewsResponse>
}
